src/MeshKernelNET/Api/IMeshKernelApi.cs

```diff
@@ -1574,6 +1590,18 @@ public interface IMeshKernelApi : IDisposable
         int PolygonRefine(int meshKernelId, in DisposableGeometryList disposableGeometryListIn, int firstIndex,

```

Comment:
I think the equidistant methods should be renamed to make it explicit that they have a different algorithm for refinement.